### PR TITLE
Fix pause on exceptions

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1710 
### Test Plan

use [exceptions](https://devtools-html.github.io/debugger-examples/examples/exceptions.html)
- [x] test each three modes, refresh each page

### CODE REVIEW

**Comment by bomsy** [comment]
Slightly confused about the issue, can't seem to replicate it. Any STR? to test the PR.

**Comment by jasonLaster** [comment]
Sorry @bomsy 

STR:
1. click on the POE button, refresh the debugger and see if it works. If you're supposed to pause on caught exceptions, does it... I used the exceptions examples page and just refreshed that.

**Comment by bomsy** [comment]
ahhh ... ok cool testing now! :)

File: src/reducers/pause.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ const State = makeRecord(({
   frames: undefined,
   selectedFrameId: undefined,
   loadedObjects: I.Map(),
-  shouldPauseOnExceptions: prefs.pauseOnExceptions,
-  shouldIgnoreCaughtExceptions: prefs.ignoreCaughtExceptions,
+  shouldPauseOnExceptions: false,
+  shouldIgnoreCaughtExceptions: false,
   expressions: I.List()
```
Comment (bomsy) [inline src/reducers/pause.js]: On testing it seems like this change is causing it to lose state on pause.
See this
foe

Comment (bomsy) [inline src/reducers/pause.js]: heya ... do you mind if i push the quick fix?

Comment (bomsy) [inline src/reducers/pause.js]: Ignore my last request ... so i don't mess the branch up  :)

**Comment by jasonLaster** [comment]
Nah. Go for it!

**Comment by bomsy** [comment]
LGTM!

**Comment by jasonLaster** [comment]
Thanks for the assist @bomsy!

**Pull request merged**